scale/docs/rest/job_execution.rst

```diff
@@ -264,10 +264,6 @@ These services provide access to information about "all", "currently running" an
 +----------------------+-------------------+--------------------------------------------------------------------------------+
 | results              | JSON Object       | An interface description for all the possible job results meta-data.           |
 +----------------------+-------------------+--------------------------------------------------------------------------------+
-| current_stdout_url   | URL               | The URL of the standard output log for the execution.                          |
-+----------------------+-------------------+--------------------------------------------------------------------------------+
-| current_stderr_url   | URL               | The URL of the standard error log for the job execution.                       |
-+----------------------+-------------------+--------------------------------------------------------------------------------+
```

Comment:
We should keep these fields in the documentation until they are actually removed from the API. You could just add a "(deprecated)" to the field descriptions.